### PR TITLE
Loki: Add step input field validation

### PR DESCRIPTION
**Pull request by ivanahuckova**
Part of https://github.com/grafana/grafana/issues/58337

This PR adds a simple query step validation to make it:

- Clear when user inputs invalid step that the step is invalid
- Provide guidance on what are examples of valid step

image
image
image
image

Part of https://github.com/grafana/grafana/issues/58337

### CODE REVIEW

File: public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx [hunk 1]
```diff
@@ -175,7 +190,7 @@ function getCollapsedInfo(query: LokiQuery, queryType: LokiQueryType, maxLines:
 
   if (!isLogQuery) {
     if (query.step) {
-      items.push(`Step: ${query.step}`);
+      items.push(`Step: ${isValidStep ? query.step : 'Invalid step'}`);
```
Comment (matyax) [inline public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx:193]: I don't like a lot reading "Step: Invalid step" (2x step). Maybe:

```suggestion
      items.push(`Step: ${isValidStep ? query.step : 'Invalid value'}`);
```

**Review by matyax — COMMENTED** [review]
Left a suggestion, but othewise LGTM!

File: public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx (hunk 1 above)
Comment (ivanahuckova) [reply public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx]: Good point! Updated.

**Review by matyax — APPROVED** [review]
🏆

**Pull request merged**